RELEASE.rst

```diff
@@ -71,26 +71,21 @@ permissions.
 * Commit these changes using ``git commit``, with a commit message like
   ``Preparing release v0.0.1``.
 
-* Run ``git clean -fxd`` to be sure you are building in a clean environment.
```

Comment:
Is it not still a good idea to clean the environment?

Reply:
Since Meson requires to build out of tree builds, built files from one build cannot contaminate another build. And meson has extremely accurate out of date detection, so you should never have to wipe your build directory and recreate it in order to trust that the correct version of the code was built.

Reply:
OK, if this is basically guaranteed to be OK, then keeping it simpler sounds good to me!